groups/prometheus/module-security-groups/main.tf

```diff
@@ -0,0 +1,42 @@
+# Security group for the Prometheus servers
```

Comment:
As above, remove the comments from this file as the resources are self descriptive.